hy/lex/__init__.py

```diff
@@ -181,8 +186,12 @@ def mangle(s):
 
 def unmangle(s):
     """Stringify the argument and try to convert it to a pretty unmangled
-    form. This may not round-trip, because different Hy symbol names can
-    mangle to the same Python identifier. See :ref:`Hy's mangling rules <mangling>`.
+    form. See :ref:`Hy's mangling rules <mangling>`.
+
+    Unmangling may not round-trip, because different Hy symbol names can mangle
+    to the same Python identifier. In particular, Python itself already
+    considers distinct strings that have the same normalized form (according to
+    NFKC) to be the same identifier.
```

Comment:
It would be nice to see an example of what this is saying

Reply:
Done.